db-service/lib/SQLService.js

```diff
@@ -170,7 +170,7 @@ class SQLService extends DatabaseService {
       // REVISIT: this is a hack the target of $now might not be a timestamp or date time
       // Add input converter to CURRENT_TIMESTAMP inside views using $now
       if(/^CREATE VIEW.* CURRENT_TIMESTAMP[( ]/is.test(query)) {
-        query = query.replace(/CURRENT_TIMESTAMP/gi, 'ISO(CURRENT_TIMESTAMP)')
+        query = query.replace(/CURRENT_TIMESTAMP/gi, "STRFTIME('%Y-%m-%dT%H:%M:%fZ','NOW')")
```

Comment:
What's the difference between `CURRENT_TIMESTAMP` and `NOW`?

The `ISO` function is a js plugin and simply creates a `new Date().toISOString()`

Reply:
`CURRENT_TIMESTAMP` only has s precision, afaik

Reply:
> The `ISO` function is a js plugin and simply creates a `new Date().toISOString()`

yes, see addition to description for alternative solution by modifying the `ISO` function.